src/main/java/duke/bin/TaskList.java

```diff
@@ -0,0 +1,79 @@
+package duke.bin;
+
+import duke.bin.common.DukeException;
+import duke.bin.task.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Contains the list of tasks with operations to add, delete, etc.
+ */
+public class TaskList {
+    private ArrayList<Task> textStorage = new ArrayList<>();
```

Comment:
It's better to put the initialization part into the constructor

Reply:
Good catch